DiscordChatExporter.Core/Discord/Data/Channel.cs

```diff
@@ -11,19 +11,22 @@ public partial record Channel(
     Snowflake Id,
     ChannelKind Kind,
     Snowflake GuildId,
+    Snowflake ParentId,
+    string? ParentName,
+    int? ParentPosition,
     ChannelCategory Category,
```

Comment:
We won't need `ChannelCategory` now that we have `ParentId`/`ParentName`, right?

Reply:
That's the idea, yeah. However, at the moment I'm actually still utilizing it to resolve `ParentId`, `ParentName` and `ParentPosition` - it's a bit of a hack right now. I'll try to dig into actually figuring out how to remove it entirely, but that may be a bit beyond me at this stage unfortunately.

Reply:
Okay, no worries, I'll look into it